hipcub/include/hipcub/rocprim/util_ptx.hpp

```diff
@@ -183,7 +183,7 @@ auto unsigned_bit_extract(UnsignedBits source,
     -> typename std::enable_if<sizeof(UnsignedBits) == 8, unsigned int>::type
 {
     #ifdef __HIP_PLATFORM_HCC__
```

Comment:
`__HIP_PLATFORM_HCC__` is also defined on clang with HIP backend?

Reply:
Seems like it, I ran into an error for hc::__bitextract_u64 without this change

Reply:
Those are HSA functions. In HIP-Clang they are just in global namespace?

Reply:
Does `__bitextract_u64` work when using `hcc`?

Reply:
Good point, thanks I will change that